drivers/gpio/gpio_dw.c

```diff
@@ -243,6 +243,9 @@ static inline void dw_pin_config(const struct device *port,
 
 	/* Set init value then direction */
 	pin_is_output = (flags & GPIO_OUTPUT) != 0U;
+
+	dw_set_bit(port_base_addr, dir_port, pin, pin_is_output);
```

Comment:
Is this the port direction register?

Reply:
@cfriedt Yes, it is the port direction register. This change basically sets the direction to output first, and then set/clr the respective pin, instead of other way round.